constants/Contributors.json

```diff
@@ -62,6 +62,9 @@
     },
     "Vixid": {},
     "Wambux": {},
+    "whalker": {
+      "suffix": "§Z§l:D"

```

Comment:
idk how i feel about Skyhanni specific chroma here

Reply:
Fair, Hanni's is the one who suggested it. If it is user disabled, it'll show grey.